### ISSUE TITLE
CNTK layer dropout

### ISSUE THREAD

**Issue by christopher5106** [Microsoft/CNTK#3510]
Hi,

I would like to implement a layer dropout. For that, I need a random number generator ops and a switch operator. Here is the implement in tensorflow:

        pred = tf.random_uniform([]) < self.dropout
        x_train = tf.cond(pred, lambda: residual, lambda: tf.nn.dropout(x, 1.0 - self.dropout) + residual)

How can I achieve the same in CNTK ?

Thank you for your answer

**Comment by acifonelli** [comment]
According to the, still opened, issue #1371 we can not random sampling from a distribution in CNTK.
